docs/domain-tenant.html

```
@@ -186,15 +495,15 @@ <h1>Tenant</h1>
 <div class="row">
     <div class="col-md-8">
         
-        <h1 id="overview">Overview <a id="overview" href="#overview" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Overview"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h1>
+        <h1 id="overview" class="content-heading">Overview <a id="overview" href="#overview" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Overview"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h1>
```

HTMLHint (1.9.2)

[error] The id value [ overview ] must be unique.

(id-unique)